Documentation/user-guides/prometheus-agent.md

```diff
@@ -0,0 +1,175 @@
+---
+weight: 151
+toc: true
+title: Prometheus Agent
+menu:
+    docs:
+        parent: user-guides
+lead: ""
+images: []
+draft: false
+description: Guide for running Prometheus in Agent mode
+---
+
+Since Prometheus-Operator `v0.64.0` it is also possible to run Prometheus in Agent mode.
+As mentioned in [Prometheus's blog](https://prometheus.io/blog/2021/11/16/agent/), Prometheus Agent
+is a deployment model optimized for environments where all collected data is forwarded to
+a long-term storage solution, e.g. Cortex or Thanos, and local Prometheus do not need storage or rule evaluation.
+
+To make it easier to upgrade from an older version to `v0.64.0` or higher, the operator is capable of running without
+the permissions to reconcile the PrometheusAgent CR. So to use the new CR, make sure to update the operator permissions to allow reconciliations of the new CR.
+
+```yaml mdox-exec="cat example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/name: prometheus-operator
+    app.kubernetes.io/version: 0.65.1
+  name: prometheus-operator
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - alertmanagers
+  - alertmanagers/finalizers
+  - alertmanagers/status
+  - alertmanagerconfigs
+  - prometheuses
+  - prometheuses/finalizers
+  - prometheuses/status
+  - prometheusagents
+  - prometheusagents/finalizers
+  - prometheusagents/status
+  - thanosrulers
+  - thanosrulers/finalizers
+  - thanosrulers/status
+  - scrapeconfigs
+  - servicemonitors
+  - podmonitors
+  - probes
+  - prometheusrules
+  verbs:
+  - '*'
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - secrets
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - services/finalizers
+  - endpoints
+  verbs:
+  - get
+  - create
+  - update
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
+  - watch
+```
+
+Similarly to Prometheus, Prometheus Agent will require permission to scrape targets. Because of this, we will create a new service account for the Agent with the necessary permissions to scrape targets.
```

Comment:
do we really need a new service account and clusterrole?

Reply:
Yeah, I guess the service account/clusterrole from the getting started guide works too. Should we not mention that at all?